dev/MRTCore/mrt/Core/src/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Windows.ImplementationLibrary" version="1.0.200519.2" targetFramework="native" />
+  <package id="TAEF.Redist.Wlk" version="10.57.200918006-develop" targetFramework="native" />
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.200729.8" targetFramework="native" />
```

Comment:
should not change this file.

Reply:
I think if we add a nuget feed/config for helix dependencies we can remove this and add that there. This would also clean up the fragile path that includes the TAEF redist version in it.

Reply:
you picked wrong packages.config

---
In reply to: [520104696](https://github.com/microsoft/ProjectReunion/pull/282#discussion_r520104696) [](ancestors = 520104696)

Reply:
Thanks I fixed this.